README.md

```diff
@@ -41,6 +41,11 @@ Note: Application keys get the same permissions as you. You might want to narrow
 
 ## If you already have a Keptn cluster running
 1. Install datadog
+
+Add datadog helm repo:
+```bash
+helm repo add datadog https://helm.datadoghq.com
+```
 ```bash
```

Comment:
nit: Can we add a line saying `Install datadog helm chart:` above this? This and we should be good.

Reply:
yes sure

Reply:
@tarangverma sorry [I might have confused you](https://github.com/keptn-sandbox/datadog-service/pull/112/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R43). I wanted `Install datadog helm chart` above line 49. Something like this:
````
1. Install datadog 

Add datadog helm repo:
```bash
helm repo add datadog https://helm.datadoghq.com
```
Install datadog helm chart: 
```bash
...
````